### DISCUSSION TITLE
DAI 2024-2025 - Practical work 3 - Drin Racaj, Nils Donatantonio, Esteban Giorgis

### DISCUSSION THREAD

**Discussion by EstebanGiorgis** [heig-vd-dai-course/heig-vd-dai-course#551] [Show and tell]
Le but est de réaliser une application web pour la gestion des notes (scolaire), avec calcul prévisionnel des moyennes et gestion des utilisateurs.

Exemple:
- Un utilisateur pourra se register/login
- Un utilisateur pourra calculer sa moyenne de X branche

**Comment by ludelafo** [comment]
Tout bon pour nous, merci !

**Comment by EstebanGiorgis** [comment]
End of the project

Here is the hash code of the latest commit : 1ad9745de0905b9c1c356b4438d587d620848f72

**Reply by ludelafo** [reply]
Hello @EstebanGiorgis,

I don't find the URL to your repository, where can I find it?

**Reply by ludelafo** [reply]
Thank you @EstebanGiorgis for the invitation that I got by email. I wasn't able to find the repository in this very discussion that's why I asked you here. :)

I'm trying to access amifucked.duckdns.org but it seems not web server is running, is it expected or am I fucked?

Thank you in advance,
Ludovic

**Reply by d0nflamingo** [reply]
Sorry my bad, forgot to turn it on again after my last try to save the ship from sinking

Should be up now, still with the certificate error

**Reply by ludelafo** [reply]
Thanks! But I'm still not able to access <https://traefik.amifucked.duckdns.org/> nor <https://amifucked.duckdns.org/users> (Page Not Found). Is the problem on my side? Are you able to access these websites?

**Reply by d0nflamingo** [reply]
End of semester tiredness didn't help with my dumbness, I still left the domain I got this morning in the environment variables. traefik.amifucked.duckdns.org is now accessible and same for the /users (obviously returning that the endpoint doesn't exist as it is a GET method, and this endpoint is configured on a POST)

**Reply by ludelafo** [reply]
Perfect, thank you! And good luck for the end of the semester.
